<!DOCTYPE html>
<!--
    So Simple Jekyll Theme 3.1.3
    Copyright 2013-2019 Michael Rose - mademistakes.com | @mmistakes
    Free for personal and commercial use under the MIT license
    https://github.com/mmistakes/so-simple-theme/blob/master/LICENSE
-->
<html lang="en-US" class="no-js">
  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  

  
    <title>Research Topics</title>
    <meta name="description" content="The University of Washington, Seattle">
    <link rel="canonical" href="https://steinbrennerlab.org/research/">
  

  <script>
    /* Cut the mustard */
    if ( 'querySelector' in document && 'addEventListener' in window ) {
      document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + 'js';
    }
  </script>

  

	<link rel="stylesheet" href="/assets/css/main.css">
  
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,400i,700,700i|Lora:400,400i,700,700i">
  <link rel="alternate" type="application/atom+xml" title="Steinbrenner Lab" href="/feed.xml">
<!-- start custom head snippets -->

<!-- insert favicons. use http://realfavicongenerator.net/ -->

<!-- end custom head snippets -->

</head>

  <body class="layout--page  research-topics">
    <nav class="skip-links">
  <h2 class="screen-reader-text">Skip links</h2>
  <ul>
    <li><a href="#primary-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
    <li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
    <li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
  </ul>
</nav>

    
  <div class="navigation-wrapper">
    <a href="#menu-toggle" id="menu-toggle">Menu</a>
    <nav id="primary-nav" class="site-nav animated drop">
      <ul><li><a href="/research/">Research</a></li><li><a href="/news/">News</a></li><li><a href="/people/">People</a></li><li><a href="/mentorship/">Mentorship</a></li><li><a href="/publications/">Publications</a></li><li><a href="/contact/">Contact & Join</a></li></ul>
    </nav>
  </div><!-- /.navigation-wrapper -->


    <header class="masthead">
  <div class="wrap">
    
      <a href="/" class="site-logo" rel="home" title="Steinbrenner Lab">
        <img src="/images/logo.jpg" class="site-logo-img animated fadeInDown" alt="Steinbrenner Lab">
      </a>
    
    
    
      
        <div class="site-title animated fadeIn"><a href="/">Steinbrenner Lab</a></div>
      
      <p class="site-description animated fadeIn" itemprop="description">The University of Washington, Seattle</p>
    
  </div>
</header><!-- /.masthead -->


    <main id="main" class="main-content" aria-label="Content">
  <article>
    

    <div class="page-wrapper">
      <header class="page-header">
        
        
          <h1 id="page-title" class="page-title">Research Topics</h1>
        
      </header>
      <div class="page-content">
        <p><br /></p>
<h2 id="plant-immunity">Plant Immunity</h2>

<p>Our lab is broadly interested in plant immune recognition of diverse pests. Different plants recognize different aspects of wounding, pathogenicity, or herbivory – how do plants balance these immunogenic inputs and activate appropriate responses? How do these responses evolve across populations and species? We leverage genetic and genomic tools in both model systems (tobacco, Arabidopsis) and non-model legume species to address these questions.</p>

<p><img src="/images/research0.png" class="align-left" alt="" /></p>

<p><br />
<br /></p>

<h2 id="immune-recognition-of-caterpillars">Immune recognition of caterpillars</h2>

<p>Plants detect pests and pathogens through cell surface receptors, which bind extracellular ligands and initiate defense signal transduction.  Many well-characterized receptors detect pathogen-associated molecular patterns (PAMPs), but receptors for other pest classes such as chewing herbivores are not well understood.</p>

<p>We recently identified the first cell surface receptor to directly detect a ligand directly associated with chewing heribivory, an 11-aa peptide termed inceptin (<a href="https://www.pnas.org/content/early/2020/11/20/2018415117">Steinbrenner et al. 2020, PNAS </a>). Inceptin receptor, or INR, confers binding, signaling, and defensive outputs in response to low nanomolar concentrations of inceptin peptide.  INR is unique to specific legume species.  Since inceptins are widespread in caterpillar oral secretions, INR likely confers broad spectrum recognition of herbivores.
<img src="/images/research00.png" class="align-left" alt="" /></p>

<p><br />
<br /></p>

<h2 id="pest-recognition-and-differentiation-by-plant-immune-receptors">Pest recognition and differentiation by plant immune receptors</h2>

<p><img src="/images/research01.png" class="align-left" alt="" width="400" />
Plant cell surface receptors signal through a network of co-receptor and adaptor kinases.  Canonical receptors recruit the co-receptor BAK1 and kinase cascades lead directly to early immune signaling outputs, such as MAPK phosphorylation, ethylene biosynthesis, and burst of reactive oxygen species.</p>

<p>Since distinct hormones mediate defense against pathogens vs herbivores, we are interested in downstream signaling differences between INR and other cell surface receptors.</p>

<p><br clear="left" /></p>

<h2 id="herbivore-resistance-in-legumes">​Herbivore resistance in legumes</h2>
<p><img src="/images/research02.png" class="align-left" alt="" width="400" />
Species in the the legume family (Fabaceae) are uniquely able to respond to inceptins.  In cowpea and common bean, inceptin activates canonical anti-herbivore defense responses such as protease inhibition, small molecule toxin biosynthesis, and release of parasitoid-attracting volatiles.</p>

<p>We are interested in the ecological functions of inceptin-triggered responses, especially in relation to the generic wound response.</p>

<p><br clear="left" /></p>

<h2 id="evolution-of-immune-receptor-function">Evolution of immune receptor function</h2>
<p><img src="/images/research3.png" class="align-left" alt="" />
Only a subset of surface receptors have immune functions; others serve critical roles in growth and development.  How do new immune functions emerge from ancestral roles for receptors?  Can we identify immune receptors from genome sequence data?</p>

<p>The INR receptor is restricted to certain legume species, but its function is idiosyncratic across species.  Soybean, for example, cannot respond to inceptin despite the presence of receptor homologs.  Common bean, on the other hand, displays novel inceptin recognition capability relative to cowpea.  Long-read sequencing may allow rapid interrogation of the INR receptor locus across populations and species to better understand evolution of herbivore recognition in legumes.  Advancing technologies in legumes will also facilitate reverse genetics to understand how INR contributes to herbivore resistance in nature.</p>

<p>Immune receptors are also useful agricultural traits. Since a given receptor is almost always unique to a given species, genus, or family, they are broadly transferrable as resistance genes. We hope to leverage the deluge of pangenomic resources in plants to facilitate rapid trait discovery.</p>

<h2 id="properties-and-utility-of-lrr-ectodomain-networks">Properties and utility of LRR ectodomain networks</h2>

<p>In constrast to analogous Toll-like receptors in animals, plant immune-related LRR sensors form a highly diversified network of heterotypic ectodomain interactions (Smakowska-Luzan 2018 doi:10.1038/nature25184). We are working to develop orthogonal ligand-sensing systems across plant and non-plant models for biotechnological use in both agriculture and medicine.</p>


        
      </div>
    </div>
  </article>
</main>


    <footer id="footer" class="site-footer">
  <!-- start custom footer snippets -->

<!-- end custom footer snippets -->
<div class="social-icons"><a class="social-icon" href="https://twitter.com/adsteinbrenner"><i class="fab fa-twitter-square fa-2x" title="Twitter"></i></a><a class="social-icon" href="https://github.com/steinbrennerlab"><i class="fab fa-github-square fa-2x" title="GitHub"></i></a></div><div class="copyright">
    
      <p>Copyright 2019, Steinbrenner Lab</p>

    
  </div>
</footer>

    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
  <script src="/assets/js/main.min.js"></script>
  <script src="https://use.fontawesome.com/releases/v5.0.12/js/all.js"></script>


  </body>

</html>
