<!DOCTYPE html>
<!--
    So Simple Jekyll Theme 3.1.3
    Copyright 2013-2019 Michael Rose - mademistakes.com | @mmistakes
    Free for personal and commercial use under the MIT license
    https://github.com/mmistakes/so-simple-theme/blob/master/LICENSE
-->
<html lang="en-US" class="no-js">
  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  
  

  
    <title>Research Topics</title>
    <meta name="description" content="The University of Washington, Seattle">
    <link rel="canonical" href="https://steinbrennerlab.org/research/">
  

  <script>
    /* Cut the mustard */
    if ( 'querySelector' in document && 'addEventListener' in window ) {
      document.documentElement.className = document.documentElement.className.replace(/\bno-js\b/g, '') + 'js';
    }
  </script>

  

	<link rel="stylesheet" href="/assets/css/main.css">
  
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,400i,700,700i|Lora:400,400i,700,700i">
  <link rel="alternate" type="application/atom+xml" title="Steinbrenner Lab" href="/feed.xml">
<!-- start custom head snippets -->

<!-- insert favicons. use http://realfavicongenerator.net/ -->

<!-- end custom head snippets -->

</head>

  <body class="layout--page  research-topics">
    <nav class="skip-links">
  <h2 class="screen-reader-text">Skip links</h2>
  <ul>
    <li><a href="#primary-nav" class="screen-reader-shortcut">Skip to primary navigation</a></li>
    <li><a href="#main" class="screen-reader-shortcut">Skip to content</a></li>
    <li><a href="#footer" class="screen-reader-shortcut">Skip to footer</a></li>
  </ul>
</nav>

    
  <div class="navigation-wrapper">
    <a href="#menu-toggle" id="menu-toggle">Menu</a>
    <nav id="primary-nav" class="site-nav animated drop">
      <ul><li><a href="/news/">News</a></li><li><a href="/research/">Research</a></li><li><a href="/people/">People</a></li><li><a href="/mentorship/">Mentorship</a></li><li><a href="/teaching/">Teaching</a></li><li><a href="/publications/">Publications</a></li><li><a href="/contact/">Contact & Join</a></li></ul>
    </nav>
  </div><!-- /.navigation-wrapper -->


    <header class="masthead">
  <div class="wrap">
    
      <a href="/" class="site-logo" rel="home" title="Steinbrenner Lab">
        <img src="/images/logo.jpg" class="site-logo-img animated fadeInDown" alt="Steinbrenner Lab">
      </a>
    
    
    
      
        <div class="site-title animated fadeIn"><a href="/">Steinbrenner Lab</a></div>
      
      <p class="site-description animated fadeIn" itemprop="description">The University of Washington, Seattle</p>
    
  </div>
</header><!-- /.masthead -->


    <main id="main" class="main-content" aria-label="Content">
  <article>
    

    <div class="page-wrapper">
      <header class="page-header">
        
        
          <h1 id="page-title" class="page-title">Research Topics</h1>
        
      </header>
      <div class="page-content">
        <p><br /></p>

<h2 id="our-big-questions">Our big questions</h2>
<ul>
  <li>
    <p>How do plants first detect attack by chewing insects? We know that plants revamp hormonal and defensive chemistry after attack, but <b>what are the molecular events in the first minutes and seconds</b> that activate these changes?
<br /></p>
  </li>
  <li>
    <p>How does the plant innate immune system evolve? We focus on <b>cell surface receptors at the plasma membrane</b>, which form a massively diverse gene family in plants underlying pest and pathogen resistance</p>
  </li>
</ul>

<h2 id="1-plant-immunity">1. Plant Immunity</h2>

<p>Our lab is interested in function and evolution of plant immune recognition. Our lens on this topic is gene families encoding pattern recognition receptors (PRRs) at the plasma membrane, which detect extracellular immunogenic molecular patterns from attackers. Using genetic and genomic tools in both molecular model systems (tobacco, Arabidopsis) and in crop legumes, we can learn which patterns are detected by different plants, and how PRR activation leads to effective resistance against pests and pathogens.</p>

<p><img src="/images/research0.png" class="align-left" alt="" width="600" />
<br clear="left" />
<br /></p>

<h2 id="2-immune-recognition-of-caterpillars">2. Immune recognition of caterpillars</h2>
<p><img src="/images/research00.png" class="align-left" alt="" width="400" /></p>

<p>PRRs bind extracellular ligands and initiate defense signal transduction. Many well-characterized receptors detect pathogen-associated molecular patterns (PAMPs), but receptors for other pest classes such as chewing herbivores are not well understood.</p>

<p>We recently identified a PRR which directly detects a peptide ligand associated with chewing heribivory. Inceptin receptor, or INR, confers binding, signaling, and defensive outputs in response to low nanomolar concentrations of inceptin peptide.  INR is unique to specific legume species.  Since inceptin peptides are widespread in caterpillar oral secretions, INR likely confers broad spectrum recognition of herbivores.</p>

<p><br clear="left" />
<b>Our papers on this topic:</b></p>
<ul>
  <li>Steinbrenner et al. 2020, “<a href="https://drive.google.com/file/d/1LFd_q5M06H4S_DCx-RX5kv_cgbU6BzLm/view">A receptor-like protein mediates plant immune responses to herbivore-associated molecular patterns</a>”, PNAS</li>
  <li>Snoeck et al. 2022, “<a href="https://drive.google.com/file/d/1CgDlij3QT4rpkGMZuLhTzW8r1GtmzKlr/view?usp=sharing">Molecular tug-of-war: Plant immune recognition of herbivory</a>”, Plant Cell</li>
</ul>

<p><br clear="left" />
<br /></p>

<h2 id="3-pest-recognition-and-differentiation-by-plant-immune-receptors">3. Pest recognition and differentiation by plant immune receptors</h2>

<p><img src="/images/research01-2.png" class="align-left" alt="" width="400" />
Plant cell surface receptors signal through a network of co-receptor and adaptor kinases.  Canonical receptors recruit the co-receptor BAK1 and kinase cascades lead directly to early immune signaling outputs, such as MAPK phosphorylation, ethylene biosynthesis, and burst of reactive oxygen species.</p>

<p>Since distinct hormones mediate defense against pathogens vs herbivores, we are interested in downstream signaling differences between INR and other cell surface receptors, especially the large family of non-kinase Receptor-like Proteins.</p>

<p><br clear="left" />
<b>Our papers on this topic:</b></p>
<ul>
  <li>A new tool for studying receptor outputs: Garcia et al. 2022, “<a href="https://www.biorxiv.org/content/10.1101/2022.07.28.501876v1">Bringing Plant Immunity to Light:  A Genetically Encoded, Bioluminescent Reporter of Pattern Triggered Immunity in Nicotiana benthamiana</a>”, bioRxiv</li>
  <li>Steinbrenner 2020, “<a href="https://drive.google.com/file/d/1l3af6XfCfEOiOEakrEQ5GFnrA5SI46M4/view?usp=sharing">The evolving landscape of cell surface pattern recognition across plant immune networks</a>”, Curr. Opin. Plant Biol.</li>
</ul>

<h2 id="4-herbivore-resistance-in-legumes">​4. Herbivore resistance in legumes</h2>
<p><img src="/images/research02.png" class="align-left" alt="" width="400" />
Species in the the legume family (Fabaceae) are uniquely able to respond to inceptins. In cowpea and common bean, inceptin activates canonical anti-herbivore defense responses such as protease inhibition, small molecule toxin biosynthesis, and release of parasitoid-attracting volatiles.</p>

<p>We are interested in the ecological functions of inceptin-triggered responses, especially in relation to the generic wound response. We are developing genetic tools in legumes, including soybean, to understand the contribution of both wounding and PRRs to resistance against herbivores.</p>

<p><br clear="left" />
<b>Our papers on this topic:</b></p>

<ul>
  <li>Steinbrenner et al. 2021, “<a href="https://www.biorxiv.org/content/10.1101/2021.09.02.458788v1.full">Signatures of plant defense response specificity mediated by herbivore-associated molecular patterns in legumes</a>”, bioRxiv (in press at Plant Journal)</li>
</ul>

<p><br clear="left" />
<br /></p>

<h2 id="5-evolution-of-immune-receptor-function">5. Evolution of immune receptor function</h2>
<p><img src="/images/research05.png" class="align-left" alt="" width="400" />
Only a subset of surface receptors are PRRs; others serve critical roles in growth and development. How do new immune functions emerge from ancestral functions?  Can we identify useful immune receptors based on genome-level features, especially given the recent deluge of plant pangenomes?</p>

<p>The INR receptor is restricted to certain legume species, but its function is idiosyncratic across species.  Soybean, for example, cannot respond to inceptin despite the presence of receptor homologs. We are sequencing additional legume genomes to understand the emergence of INR across 50 million years of legume evolution. We are also transferring INR as a resistance trait to herbivore-susceptible crop species.</p>

<p><br clear="left" />
<b>Our papers on this topic:</b></p>
<ul>
  <li>Snoeck et al. 2022, “<a href="https://www.biorxiv.org/content/10.1101/2022.03.30.484633v1.full">Evolutionary gain and loss of a plant pattern-recognition receptor for HAMP recognition</a>”, bioRxiv and in press at ELIFE</li>
  <li>Schultink and Steinbrenner 2021, “<a href="https://drive.google.com/file/d/1aLeg2RQXgjAUeg3xWW1sNg6kYbBw_xPs/view?usp=sharing">A playbook for developing disease-resistant crops through immune receptor identification and transfer</a>”, Curr. Opin. Plant Biol.</li>
  <li>Snoeck et al. 2023, “Plant Receptor-like proteins (RLPs): Structural features enabling versatile immune recognition.” Physiological and Molecular Plant Pathology. <a href="https://authors.elsevier.com/a/1gprb39MrQ4oiE">(link)</a></li>
</ul>

<p><br clear="left" />
<br /></p>

<h2 id="6-properties-and-utility-of-lrr-ectodomain-networks">6. Properties and utility of LRR ectodomain networks</h2>

<p>In contrast to analogous Toll-like receptors in animals, which contain a structurally similar leucine-rich repeat (LRR) ectodomain, plant immune-related LRR sensors form a highly diversified network of heterotypic ectodomain interactions (<a href="https://www.nature.com/articles/nature25184">Smakowska-Luzan 2018 doi:10.1038/nature25184</a>). We are working to develop orthogonal ligand-sensing systems across plant and non-plant models for biotechnological use in both agriculture and medicine.</p>


        
      </div>
    </div>
  </article>
</main>


    <footer id="footer" class="site-footer">
  <!-- start custom footer snippets -->

<!-- end custom footer snippets -->
<div class="social-icons"><a class="social-icon" href="https://twitter.com/adsteinbrenner"><i class="fab fa-twitter-square fa-2x" title="Twitter"></i></a><a class="social-icon" href="https://github.com/steinbrennerlab"><i class="fab fa-github-square fa-2x" title="GitHub"></i></a></div><div class="copyright">
    
      <p>Copyright 2022, Steinbrenner Lab</p>

    
  </div>
</footer>

    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=" crossorigin="anonymous"></script>
  <script src="/assets/js/main.min.js"></script>
  <script src="https://use.fontawesome.com/releases/v5.0.12/js/all.js"></script>


  </body>

</html>
